### PR TITLE
w2f

### PR DESCRIPTION
**Pull request by msieb1**

### CODE REVIEW

File: vllm/model_executor/models/falcon.py [hunk 1]
```diff
@@ -193,34 +239,48 @@ class FalconMLP(nn.Module):
...
     def forward(self, x: torch.Tensor) -> torch.Tensor:
         # NOTE(zhuohan): Following huggingface, we do not fuse bias add here.
         x, bias = self.dense_h_to_4h(x)
         if bias is not None:
             x += bias
+        x = torch.clamp(x, -4, 4.)
+        self.logt('middle_in/clamped_middle_in', x)
+
         x = self.act(x)
+        x = torch.clamp(x, -16., 16.)
```
Comment (sshleifer) [inline vllm/model_executor/models/falcon.py:281]: can delete this since we already clamped before

File: vllm/model_executor/models/falcon.py [hunk 2]
```diff
@@ -279,16 +345,27 @@ def forward(
...
                 residual += attention_output
-                mlp_layernorm_out = self.post_attention_layernorm(residual)
+                # mlp_layernorm_out = self.post_attention_layernorm(residual) # WHIPPET uncomment
 
         # MLP.
-        mlp_output, mlp_bias = self.mlp(mlp_layernorm_out)
+        mlp_input = self.input_layernorm(residual) # WHIPPET
+        self.logt('layer_input_1/normed_residual', mlp_input)
+
+        mlp_input = torch.clamp(mlp_input, -4., 4.) # WHIPPET
```
Comment (sshleifer) [inline vllm/model_executor/models/falcon.py:361]: we already clamp inside mlp

**Comment by sshleifer** [comment]
He doesnt want to merge it, he was just trying to show me a diff

**Comment by russellb** [comment]
> He doesnt want to merge it, he was just trying to show me a diff

based on this comment, and that this hasn't been updated in several months, I'm assuming it can just be closed. Feel free to reopen if you still need this for some reason.

**Pull request closed without merging**